### ISSUE TITLE
Enable all uBlock filters

### ISSUE THREAD

**Issue by PekaFat** [gorhill/uBlock#1533]
If I enable all filters in "3p-filters" tab, how uBlock will prioritize rules, if they will exist in multiple filter lists? uBlock will use and "see" only one, or it will be loading all duplicates of same rule?

**Comment by curiosity-seeker** [comment]
No, uB0 uses duplicate filters only once, see
https://github.com/gorhill/uBlock/wiki/Launch-and-filter-lists-load-performance
